### DISCUSSION TITLE
Alternatives for uploading documentation

### DISCUSSION THREAD

**Discussion by ams015** [community/community#87023] [GitHub Education]
### Hi everyone,

It is asking for verification for my academic institution, and my id does not show the school year so seems like some other documentation is the only option; I am trying to use my transcript. However I can only directly take a photo using my laptop camera; which only results in a blurry photo and only 1 photo. I need to be able to upload a document or upload multiple photos from my desktop.

**Comment by faridnec** [comment]
I guess one of the main transcript page would be enough. I took my ID and showing it only one face. They still approved my application. I hope this works for you too
